src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -538,6 +538,24 @@ describe('ReactDOMComponent', function() {
       expect(console.error.argsForCall[0][0]).toContain('clipPath');
       expect(console.error.argsForCall[0][0]).toContain('clip-path');
     });
+    
+    it('should only warn once about deprecated SVG attributes', function() {
+      spyOn(console, 'error');
+      var container = document.createElement('div');
+      ReactDOM.render(
+        <svg clipPath="0 0 100 100">
+          <rect strokeWidth={1} />
+          <rect strokeWidth={10} />
```

Comment:
I do actually have it just below: I wanted to check that both having two elements *and* having two renders produces a unique result per attribute name.